ui_tests/src/test/resources/com/saucedemo/account/signinNegative.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Sign In
+
+
+  @signIn
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+  @signIn
+  Scenario: Sign in with invalid user name
+    Given I open start page
+    When I set 'thisIsSomeInvalidUser' as username on Sign in page
+    And I set 'standard_password' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with empty user and password
+    Given I open start page
+    When I set ' ' as username on Sign in page
+    And I set ' ' as password on Sign in page
```

Comment:
@Arman0001  You're not setting empty username and password here, you're setting ` ` there.
I covered this in last class (8)